templates/chain.html

```diff
@@ -138,39 +138,72 @@ <h4>Facts</h4>
-                {% for adv in adversaries %}
-                    <option value="{{ adv.id }}">{{ adv.name }}</option>
-                {% endfor %}}
-              </select>
-              <select id="advFactSource" onchange="refreshColorCodes()">
-                <option value="" selected>Overlay fact source</option>
-                    {% for s in sources %}
-                        <option value="{{ s.facts | map(attribute='property') | list }}">{{ s.name }}</option>
-                    {% endfor %}
-              </select>
-              <button id="advNewBtn" type="button" class="button-notready atomic-button" onclick="saveNewAdversary()">Save new</button>
-            </div>
-            <div id="phases" class="column adversary-header" style="flex:75%;text-align: left">
-                <input id="profile-goal" type="text" class="advGoal">
-                <input id="profile-description" type="text">
-                <br><br>
-                <div id="dummy"></div>
+            <div class="row">
+                <div class="column section-border" style="flex:25%;">
+                    <img src="/chain/img/hacker.png">
+                    <h4 style="margin-bottom:-15px">Adversaries</h4>
+                    <br>
+                    <div class="toggle">
+                        <label class="switch">
+                            <input type="checkbox" id="togBtnOp" checked onchange="toggleAdversaryView()">
+                            <div class="slider round"><span class="on">VIEW</span><span class="off">NEW</span></div>
+                        </label>
+                    </div>
+                    <p class="section-description" style="margin-top: 0px">
+                        Adversaries are collections of ATT&CK TTPs, designed to test specific threats. Abilities with unmet requirements are faded out to show the adversary cannot use them unless an unlocking ability is added.
+                    </p>
+        
+                    <div id="viewAdversary">
+                        <select id="profile-existing-name" onchange="loadAdversary();">
+                            <option value="" disabled selected>Select existing</option>
+                            {% for adv in adversaries %}
+                            <option value="{{ adv.id }}">{{ adv.name }}</option>
+                            {% endfor %}}
+                        </select>
+                        <select id="advFactSource" onchange="refreshColorCodes()">
+                            <option value="" selected>Overlay fact source</option>
+                            {% for s in sources %}
+                            <option value="{{ s.facts | map(attribute='property') | list }}">{{ s.name }}</option>
+                            {% endfor %}
+                        </select>
+                        <button id="advNewBtn" type="button" class="button-notready atomic-button" onclick="saveNewAdversary()">Save / update</button>
```

Comment:
I would remove this button entirely. When viewing an adversary, you can't update it, you can only 'save it as new'. I think you capture this idea nicely through the 'new' toggle button (i.e., you can view an adversary, toggle to 'new' and save it), which would make it redundant here.

Reply:
good idea